docs/sphinx/dev/dev_style_guide_code.rst

```diff
@@ -11,8 +11,9 @@ Naming
 - Type names in kotekan should be nouns, use *CamelCase* formatting and begin
```

Comment:
While we are at it, could we also update references to `CamelCase` to `PascalCase`, since normally `camelCase` starts with a lowercase letter. 
```suggestion
- Type names in kotekan should be nouns, use *PascalCase* formatting and begin
```